docker/resource-provider/Dockerfile

```diff
@@ -28,8 +21,14 @@ RUN go build -v -ldflags="-X 'github.com/lilypad-tech/lilypad/pkg/system.Version
 ENV DISABLE_POW=true
 
 FROM build-$COMPUTE_MODE AS final
-RUN mv lilypad /usr/local/bin
+# Default environment variables
+ARG NETWORK=testnet
+ARG DISABLE_TELEMETRY=false
```

Comment:
This one was previously an ENV, but now an ARG, is that on purpose?

Reply:
> This one was previously an ENV, but now an ARG, is that on purpose?

@narbs91 good catch! and yes, it's intentional - we "bake" that in at build-time to the `run` script (see a few lines below) so there's not actually any value having it as `ENV` (something to clean up later).

Reply:
Ah gotcha, good stuff!